server/apps/content_types/content_types.py

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Well, basically you have here a resource and a service you don't need - you can use BasicService directly in __init__.py
See: https://github.com/superdesk/superdesk-content-api/tree/master/content_api/clients